app/elements/password-page/password-page.html

```diff
@@ -91,16 +91,7 @@
                     },
                     user: {
                         type: Object,
-                        value: function() {
-                            return {
-                                currentPassword: null,
-                                newPassword: null,
-                                confirmNewPassword: null,
-                                email: EMAIL,
-                                hasPass: HAS_PASS
-                            }
-                        },
-                        notify: true
+                        computed: "_computeUserPass(user)"
```

Comment:
User here is not `model.user` but a new object. We need it for dual-data binding. It has some new properties like `currentPassword` etc and some `model.user` ones.